geopandas/array.py

```diff
@@ -609,15 +612,15 @@ def distance(self, other):
         self.check_geographic_crs(stacklevel=6)
         return self._binary_method("distance", self, other)
 
-    def buffer(self, distance, resolution=16, **kwargs):
+    def buffer(self, distance, resolution=16, **kwargs) -> "GeometryArray":
```

Comment:
why not `distance` and `resolution` add typing hints?

Reply:
Where I didn't add type hints was when it wasn't clear what all types were allowed as an argument for the parameter.